concent_api/concent_api/settings/base.py

```diff
@@ -64,9 +64,10 @@
 # https://docs.djangoproject.com/en/1.11/ref/settings/#databases
 
 DATABASES = {
-    'default': {
+    'default': {},
```

Comment:
We need a comment explaining how `'default'` works now and why it's still here. Actually a comment explaning the whole issue with `DATABASES` and our assumptions about it would be best.

Reply:
:+1: DONE